### ISSUE TITLE
devtoolset-6

### ISSUE THREAD

**Issue by fabiencastan** [wwfxuk/centos-vault-scl#3]
Hi,

I tried what is suggested in the README to get access to an older devtoolset within a docker image of centos7.
```
> docker run -it centos:centos7 /bin/bash

yum install -y centos-release-scl-rh yum-utils
curl -sSL https://raw.githubusercontent.com/wwfxuk/centos-vault-scl/master/activate.sh | bash
yum search devtoolset-6
```

But I still get: `No matches found`.

Here is the output of the command:
```
# curl -sSL https://raw.githubusercontent.com/wwfxuk/centos-vault-scl/master/activate.sh | bash
/usr/bin/tar
/usr/bin/yum-config-manager
Extracting *.repo files from https://github.com/wwfxuk/centos-vault-scl/archive/master.tar.gz
into /etc/yum.repos.d...
centos-vault-scl-master/CentOS-Vault-SCLo-scl-rh.repo
centos-vault-scl-master/CentOS-Vault-SCLo-scl.repo
total 68K
drwxr-xr-x 1 root root 4.0K May 10 18:27 .
drwxr-xr-x 1 root root 4.0K May 10 18:27 ..
-rw-r--r-- 1 root root 1.7K Apr  7 22:01 CentOS-Base.repo
-rw-r--r-- 1 root root 1.3K Apr  7 22:01 CentOS-CR.repo
-rw-r--r-- 1 root root  649 Apr  7 22:01 CentOS-Debuginfo.repo
-rw-r--r-- 1 root root  630 Apr  7 22:01 CentOS-Media.repo
-rw-r--r-- 1 root root  971 Oct 29  2018 CentOS-SCLo-scl-rh.repo
-rw-r--r-- 1 root root 1.3K Apr  7 22:01 CentOS-Sources.repo
-rw-rw-r-- 1 root root 5.4K Sep 20  2019 CentOS-Vault-SCLo-scl-rh.repo
-rw-rw-r-- 1 root root 5.6K Sep 20  2019 CentOS-Vault-SCLo-scl.repo
-rw-r--r-- 1 root root 7.4K Apr  7 22:01 CentOS-Vault.repo
-rw-r--r-- 1 root root  314 Apr  7 22:01 CentOS-fasttrack.repo
-rw-r--r-- 1 root root  616 Apr  7 22:01 CentOS-x86_64-kernel.repo
Enabling C7.8.2003-centos-sclo-sclo...
```

What do I missed to get access to `devtoolset-6`?

Thanks,

**Comment by j0yu** [comment]
Hey! I've just come across this while doing #2.

It's in the final stages now before getting merged into master but if you need a working solution now, try using this

```
curl -sSL https://raw.githubusercontent.com/wwfxuk/centos-vault-scl/6d2ba4bd7c10feccc44eac59aaada891e0bb26ae/activate.sh | bash
```

Let me know if it works so I can close this issue once #2 is merged

**Comment by j0yu** [comment]
I just merged #2 into master, the following works for me now

```bash
yum install -y centos-release-scl-rh yum-utils
curl -sSL https://raw.githubusercontent.com/wwfxuk/centos-vault-scl/master/activate.sh | bash
yum search devtoolset-6
```

**Comment by fabiencastan** [comment]
Yes, it works now.
Thanks!
